rezolvari probleme.js

```diff
@@ -0,0 +1,71 @@
+//problema 1
+function SumVect(vect)
+{
+    var sum = 0;
+
+    for(i=0; i < vect.length; i++)
+    {
+        sum += vect[i];
+    }
+
+    return sum;
+}
+
+//problema 2
+function Get2LargestNo(vect)
+{
+    var numbers = [];
+    numbers[0] = -Infinity;
+    numbers[1] = -Infinity;
+
+    for(i=0; i < vect.length; i++)
+    {
+        if(numbers[1] < vect[i])
+        {
+            numbers[0] = numbers[1];
+            numbers[1] = vect[i];
+        }
+    }
+
+    return numbers;
+}
+
+//problema3
+function ThereAreDuplicate(vect)
+{
+    var suma1 = SumVect(vect);
+	//console.log('suma1 : ' + suma1);
```

Comment:
bad comments, probably used to debug the solution instead of making proper unit testing